### PR TITLE
Add elasticsearch output

### PR DESCRIPTION
**Pull request by SimonForsman**

### CODE REVIEW

**Review by joaoaav — APPROVED** [review]
I'm approving it, but i'm really not the best to tell if these changes are good and correct or not

File: fluentd-loggly/td-agent.conf [hunk 1]
```diff
@@ -324,13 +324,30 @@
...
+    buffer_path /var/log/fluentd_loggly_buffer
+    flush_interval 10s
+  </store>
+  <store>
+    @type elasticsearch
+    host "#{ENV['ELASTIC_HOST']}"
+    scheme "#{ENV['ELASTIC_SCHEME']}"
+    port "#{ENV['ELASTIC_PORT']}"
+    user "#{ENV['ELASTIC_USERNAME']}"
+    password "#{ENV['ELASTIC_PASSWORD']}"
```
Comment (niklas-r) [inline fluentd-loggly/td-agent.conf:345]: Are these Kubernetes secrets?

File: fluentd-loggly/td-agent.conf [hunk 2]
```diff
@@ -324,13 +324,30 @@
   reserve_data true
 </filter>
 
-# Send logs to Loggly
+# Send logs to Loggly and Elastic
```
Comment (niklas-r) [inline fluentd-loggly/td-agent.conf:327]: Great that we start with sending to both during the evaluation period!

**Review by niklas-r — APPROVED** [review]
Same as Joao 😂

File: fluentd-loggly/td-agent.conf (hunk 1 above)
Comment (SimonForsman) [reply fluentd-loggly/td-agent.conf:345]: No, its just part of the configuration for now

**Pull request merged**